charts/mediawiki/values.yaml

```diff
@@ -25,6 +25,8 @@ mw:
   redis:
     readServer: someRedisOtherServer
     writeServer: someRedisServer
+    readCacheServer: aThirdRedisServerForVeryVolatileKeys
```

Comment:
and this is where `redis` will be used?

Reply:
yes, exactly; think I ought to make it clearer? Any suggestion how?

Reply:
nah I just was trying to connect the dots to the other PR :)